src/components/TopBar/TopBar.tsx

```diff
@@ -83,20 +86,25 @@ export class TopBar extends React.PureComponent<ComposedProps, State> {
 
     const width = getWidth(logo, 104);
 
-    const logoMarkup = logo ? (
-      <UnstyledLink
-        url={logo.url || ''}
-        className={styles.LogoLink}
-        style={{width}}
-      >
-        <Image
-          source={logo.topBarSource || ''}
-          alt={logo.accessibilityLabel || ''}
-          className={styles.Logo}
-          style={{width}}
-        />
-      </UnstyledLink>
-    ) : null;
+    const contextMarkup =
+      !contextControl && logo ? (
```

Comment:
We will want to use if-else flow control here:

if context control -> render context control
else if logo -> render logo
else -> null

Reply:
Makes sense that makes it a bit clearer and resolves the no `contextControl` nor `logo` case.